src/test/rpc/AccountObjects_test.cpp

```diff
@@ -1220,6 +1220,164 @@ class AccountObjects_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testAccountObjectMarker()
+    {
+        testcase("AccountObjectMarker");
+
+        using namespace jtx;
+        Env env(*this);
+
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+        Account const carol{"carol"};
+        env.fund(XRP(10000), alice, bob, carol);
+
+        unsigned const accountObjectSize = 30;
+        for (unsigned i = 0; i < accountObjectSize; i++)
+            env(check::create(alice, bob, XRP(10)));
+
+        for (unsigned i = 0; i < 10; i++)
+            env(token::mint(carol, 0));
+
+        env.close();
+
+        unsigned const limit = 11;
```

Comment:
if you make it `static constexpr` ...

Reply:
never mind, you can capture `limit` also without this change. Although `constexpr` would make it more obvious.

Reply:
can I just update the lambda function capture to `[&]`, you can review in my recent commit.